config-example.yaml

```diff
@@ -87,6 +87,9 @@ derp:
     region_code: "headscale"
     region_name: "Headscale Embedded DERP"
 
+    # Verify clients to this DERP server using the Headscale node list
+    verify_clients: true
```

Comment:
We can have it "recommended" on by default here, but you need to add a CHANGELOG entry saying that this is now on by default.